docs/release-notes/release-notes-0.14.1.md

```diff
@@ -28,6 +28,11 @@
   helpers](https://github.com/lightninglabs/lightning-terminal/pull/928) that 
   allow for more control over running unit tests. 
  
+* Correctly [move a session to the Expired 
```

Comment:
v0.14.1 is already released :)! So this should be release-docs for v0.14.2.
You could copy the docs from:
https://github.com/lightninglabs/lightning-terminal/pull/974/
& specifically
https://github.com/lightninglabs/lightning-terminal/pull/974/files#diff-a414f99dddb12f8b527d87981dbb216b60f168b2c7bd360bd9bc876fa1e393b8 

If you'd prefer. I'll update that PR later today.